packages/runtime/src/state/modules/locales.ts

```diff
@@ -0,0 +1,56 @@
+import { Action, ActionTypes } from '../actions'
+
+export type Locale = string
```

Comment:
I would like us to strongly type this

Reply:
Hmm... I thought that ISO standards were comprehensive and didn't change. Maybe we don't use the unicode spec and instead use the ISO standards? ISO-639 and ISO-3166 was it?

Reply:
Yes if we just want to have `${language_code}-${country_code}` then it'll be easier to manage. But still, like the ISO-3166 is [regularly updated](https://www.iso.org/iso-3166-country-codes.html:
> ISO 3166 is regularly updated to reflect changes in country names and subdivisions. These changes are done by the ISO 3166 Maintenance Agency (ISO 3166/MA).

I was pointing to the Unicode specification because that's what the Next.js docs is using.

Reply:
Ok. Let's just do arbitrary string values, then